src/runtime_src/core/common/xclbin_parser.cpp

```
@@ -867,6 +867,9 @@ get_aie_partition(const axlf* top)
     aie_pdi_obj pdiobj;
     auto aiepdip = reinterpret_cast<const aie_pdi*>(topbase + aiep->aie_pdi.offset + i * sizeof(aie_pdi));
 
+    if (aiepdip->pdi_image.size > PDI_IMAGE_MAX_SIZE)
```

Do we want to throw an error here?